June cloud release notes (#17176)
cockroachdb/docs: src/current/_includes/releases/cloud/2023-06-05.md

## June 05, 2023

<h3> Console changes </h3>

- Organizations that have purchased premium support will now see it included in their [invoices](../cockroachcloud/billing-management.html).
- Cross-cluster [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) are now limited to {{ site.data.products.dedicated }} clusters with a major version greater than or equal to the major version of the source cluster.

> - The [**Restore jobs**](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) tab of the **Backups page** now shows more information about the restore job, such as the source and destination clusters, the restore type, the backup size, and the job progress.

Reviewer: ```suggestion
- The [**Restore jobs**](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) tab of the **Backups page** now shows more information about a restore job, such as the source and destination clusters, the restore type, the backup size, and the job's progress.
```

<h3> Security updates </h3>

> - The [Organization Audit Logs API](../cockroachcloud/cloud-org-audit-logs.html), which provides logs of events that occur in a Cloud organization, is now [generally available](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html#feature-availability-phases).

Reviewer: Oops, I completely forgot to write a release note for this. Thank you!

> - `ExternalId` support is now re-enabled in the [Customer-managed Encryption Key (CMEK)](../cockroachcloud/cmek-ops-aws.html) capability on AWS.

Reviewer: Is this worth a release note, @abhinavg6 ?

Author: I think this was added by Abhinav in the submission process

<h3> Bug fixes </h3>

> - The `status` returned by the [`logexport` Cloud API endpoint](../cockroachcloud/export-logs.html#the-logexport-endpoint) is now determined by the readiness of the underlying logging resources, in addition to the latest log export update job state. Before this change, it was possible for `GET` requests to the `logexport` to report a log export status that conflicted with the latest log export update job state or with the state of the logging infrastructure.

Reviewer: ```suggestion
- The `status` returned by the [`logexport` Cloud API endpoint](../cockroachcloud/export-logs.html#the-logexport-endpoint) is now determined by the state of both the latest log export's job state and the readiness of the underlying logging resources. Before this change, a `GET` request to the `logexport` endpoint could report an outdated log export status that conflicted with the latest log export update job state or with the most recent state of the logging infrastructure.
```

> - Fixed a bug where concurrent [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) could run on the same destination cluster.

Reviewer: Could this lead to incorrectly-restored data or just consume extra resources?

Author: looked into the pr, the description says the cluster could become "unusable" https://github.com/cockroachlabs/managed-service/pull/13194

- Fixed a bug where the IOPS price preview shown when [creating](../cockroachcloud/create-your-cluster.html) or [editing a cluster](../cockroachcloud/cluster-management.html) was inaccurate.

> - The **Group** tab is now only shown to users who have access to it.

Reviewer: ```suggestion
- The **Group** tab now appears only to users who have the `TBD` privilege.
```
Is this based on a specific privilege? I left it as TBD in my suggestion, but you should be able to edit it. Was this a privilege escalation bug that has been fixed, where people could see info they shouldn't have access to, or would the Group tab appear before but be empty if you didn't have access?

Author: the page was empty if you didn't have access, i'll follow up with eng about who has access to the page. seems like it was supposed to be behind a feature flag.

<h3> Bug fixes </h3>

- The `status` returned by the [`logexport` Cloud API endpoint](../cockroachcloud/export-logs.html#the-logexport-endpoint) is now determined by the state of both the latest log export's job state and the readiness of the underlying logging resources. Before this change, a `GET` request to the `logexport` endpoint could report an outdated log export status that conflicted with the latest log export update job state or with the most recent state of the logging infrastructure.
- Fixed a bug where concurrent [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) could run on the same destination cluster and cause the destination cluster to become unusable.
- Fixed a bug where the IOPS price preview shown when [creating](../cockroachcloud/create-your-cluster.html) or [editing a cluster](../cockroachcloud/cluster-management.html) was inaccurate.

> - The **Group** tab is now only shown to users who have this feature enabled. Previously, an error page was shown to users who navigated to the **Group** tab without access.

Reviewer: ```suggestion
- The **Group** tab is now shown only to users who have this feature enabled. Previously, an error page was shown to users who navigated to the **Group** tab without enabling the feature.
```
Moving "only" is an optional nit and I can't really explain why it seems better to me.
Changed "without access" based on your comment about what was fixed. Is it more accurate?
